deploy/charts/openebs-monitoring/templates/podmonitors.yaml

```diff
@@ -15,7 +15,12 @@ spec:
   namespaceSelector:
 {{ toYaml $fields.namespaceSelector | indent 4 }}
   podMetricsEndpoints:
-{{ toYaml $fields.podMetricsEndpoints | indent 2 }}
+    - targetPort: {{ $fields.podMetricsEndpoints.targetPort }}
```

Comment:
Extra spaces are not required. `indent 6` should work fine.

Reply:
Yes, but this is the standard way of doing. Everywhere we are following this pattern now-a-days.